website/docs/index.html.markdown

```diff
@@ -67,7 +67,7 @@ explained below:
 
 ### Static Credentials
 
-!> **Warning:** Hard-coding credentials into any Terraform configuration is not
+!> **Warning:** Hard-coding credentials into any Terraform configuration are not
```

Comment:
This update tripped me up a little bit as the verb here reads to me as attached to the singular subject "configuration" and not the plural "credentials", therefore the original "is" seems more correct to me.

```suggestion
!> **Warning:** Hard-coding credentials into any Terraform configuration is not
```

I think if the sentence was reworded a little around "Hard-coding credentials are not recommended in any Terraform configuration" or "It is not recommended to hard-code credentials into any Terraform configuration" it might remove the ambiguity. 👍

Reply:
@bflad I agree. We can reword a little **Hard-coding credentials are not recommended in any Terraform configuration** or we should proceed with the **are** I think.